CUSTOM.md

```diff
@@ -0,0 +1,218 @@
+  - adversaryUI
+    - hasUI: boolean - set to true if you have AdversaryUI function
+    - AdversaryUI(params)
+      - params: table - contains data about game setup
+        - level: number - level of current adversary
+        - supporting: boolean - whether this is supporting adversary or not
+      - return table - note setup adversary effects can be excluded from UI
+        - loss
+          - tooltip: string - text of adversary loss condition, use \\n to add line breaks
+          - counter
+            - text: string - text that should appear before counter
+            - buttons: boolean - whether buttons should appear for counter, i.e. Habsburg
+            - callback: function - name of callback to occur after loss counter buttons are pressed (only called when buttons is true)
+        - escalation
+          - tooltip: string - text of adversary escalation, use \\n to add line breaks
+          - random: boolean - whether random terrain button needs to appear for adversary (only exists for secondary adversary)
+        - one
+          - name: string - name of adversary effect
+          - tooltip: string - text of adversary effect, use \\n to add line breaks
+        - two
+          - name: string - name of adversary effect
+          - tooltip: string - text of adversary effect, use \\n to add line breaks
+        - three
+          - name: string - name of adversary effect
+          - tooltip: string - text of adversary effect, use \\n to add line breaks
+        - four
+          - name: string - name of adversary effect
+          - tooltip: string - text of adversary effect, use \\n to add line breaks
+        - five
+          - name: string - name of adversary effect
+          - tooltip: string - text of adversary effect, use \\n to add line breaks
+        - six
+          - name: string - name of adversary effect
+          - tooltip: string - text of adversary effect, use \\n to add line breaks
+      - <loss callback>(params)
+        - params: table - contains data about update
+          - count: number - new count for loss counter
+      - UpdateAdversaryLossCounter(params) - when adversary doesn't use buttons you can call this function to update text, i.e. Russia
+        - params: table - contains data about update
+          - support: boolean - whether this is supporting adversary or not
+          - count: number - new count for loss counter
+  - limitSetup - called before board setup starts in case any pieces need to be limited, i.e. France
+    - limitSetup: boolean - set to true if you have LimitSetup function
+    - LimitSetup(params)
+      - params: table - contains data about game setup
+        - level: number - level of current adversary
+        - other: table - contains data about other adversary
+          - exist: boolean - whether there is a second adversary
+          - level: number - other adversary level
+- Scenario Specific:
```

Comment:
Probably it should be stated that this used the same programing logic as adversaries.

Reply:
so everything in Adversary Specific section is not relevant for scenarios, but anything not under there is since it's under the Adversary & Scenario section